### ISSUE TITLE
default regularization parameters in the documentation

### ISSUE THREAD

**Issue by paprakash** [uf3/uf3#43]
the default value of regularization parameters might need to be changed in the documentation

**Comment by sxie22** [comment]
I agree. The current default parameters were determined from the bcc tungsten benchmarks in our preprint. Actual optimum values for a given system will depend on the total number of coefficients (b-spline resolution, number of components, ...), quantity of training data, and, probably, other factors that deserve further investigation.
